lib/rails_param/coercion/hash_param.rb

```diff
@@ -2,10 +2,14 @@ module RailsParam
   class Coercion
     class HashParam < VirtualParam
       def coerce
+        delimiter = options[:delimiter] || ","
```

Comment:
Can you extract `delimiter` and `separator` to methods? Something like
```ruby
def delimiter
  options.fetch(:delimiter, ",")
end
```

It will probably require to change this line https://github.com/nicolasblanco/rails_param/blob/master/lib/rails_param/coercion/virtual_param.rb#L6
`options: {}`